### PR TITLE
[WIP] Glide Changes 

### PR DESCRIPTION
**Pull request by ramkrsna**
- This change removes the makefiles to work with glide
- travis.yaml is using glide dependency instead of godep

### CODE REVIEW

File: client/cli/go/glide.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package: github.com/heketi/heketi/client/cli/go
+import:
+- package: github.com/heketi/heketi
```
Comment (lpabon) [inline client/cli/go/glide.yaml]: Is this package necessary?  Pulling the source?  I could see this causing issues if someone adds a file.

**Comment by lpabon** [comment]
Also, could you please put the removal of all the Godeps onto its own commit?  It makes it much easier to review.

File: client/cli/go/glide.yaml (hunk 1 above)
Comment (ramkrsna) [inline client/cli/go/glide.yaml]: Ack. Will remove this package. I have been having other issues with the Makefile. The client heketi package has its own dependencies; it is trying to create a vendor file, glide.yaml and a lock file. Any idea how to tackle that issue.

**Comment by ramkrsna** [comment]
I have considered all the suggestions and sent in a new commit in #706

**Pull request closed without merging**